src/signTxOutput.h

```diff
@@ -51,6 +53,7 @@ typedef struct {
 	uint16_t currentToken;
 	bool includeDatumHash;
```

Comment:
This should be updated: `output_context_t` must support everything needed, including the post alonzo stuff. But first let's get txhashbuilder finished.